client/components/main/README.md

```diff
@@ -16,3 +16,4 @@ function render() {
 
 - `className`: Add your own CSS class to the wrapper.
 - `wideLayout`: Use wide layout (larger `max-width`) for the section.
+- `scrollToTopOnMount`: Scroll to top when the component is mounted to avoid showing the previous scroll position.
```

Comment:
Remnant of the previous implementation?

Reply:
Yes!, removed https://github.com/Automattic/wp-calypso/pull/73783/commits/a76772696e29e8d47fa618ef3906fd496dbaef2d
Thank you!